imbi/endpoints/operations_log.py

```diff
@@ -166,3 +174,32 @@ class RecordRequestHandler(_RequestHandlerMixin,
                ticket_slug = %(ticket_slug)s,
                version = %(version)s
          WHERE id = %(id)s""")
+
+    async def delete(self, *args, **kwargs):
+        await super().delete(*args, **kwargs)
+        await self.search_index.delete_document(kwargs['id'])
+
+    async def patch(self, *args, **kwargs):
+        await super().patch(*args, **kwargs)
+        await self.index_document(kwargs['id'])
+
+
+class SearchIndexRequestHandler(operations_log.RequestHandlerMixin,
+                                base.AuthenticatedRequestHandler):
+    SQL = re.sub(r'\s+', ' ', """\
+        SELECT id
+          FROM v1.operations_log
+         ORDER BY id""")
+
+    async def get(self):
```

Comment:
This should be a `POST` instead of a `GET` since it is modifying the state of the system.  We should probably restrict this to "admin" users as well so add `@base.require_permission('admin')` to the method as well.